gitpractice02

```diff
@@ -0,0 +1,2 @@
+Hi everyone
```

Comment:
please add an exclamation mark at the end of this line. :-1:

Reply:
why

Reply:
wtf

Reply:
just for a practice

Reply:
Do it if you don't wanna be fired.

Reply:
I just checked your new commit. Fine by me.

Reply:
Then remove your red sign man